.travis.yml

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------------------------
+# Travis CI scripts
+# Copyright(c) pgRouting Contributors
+#
+# Main configuration
+# ------------------------------------------------------------------------------
+
+dist: trusty
```

Comment:
We shouldn't use Trusty because is near of his EOL (April 2019). We should use Bionic Beaver.